hack/testing/check-EFK-running.sh

```diff
@@ -0,0 +1,193 @@
+#!/bin/bash
+
```

Comment:
Guess for me it depends if you're ever going to look at the output. For tests I would expect you don't, unless it breaks.